examples/example_pitchbend/main.js

```diff
@@ -0,0 +1,161 @@
+import "./Superpowered.js";
+
+var webaudioManager = null; // The SuperpoweredWebAudio helper class managing Web Audio for us.
+var Superpowered = null; // A Superpowered instance.
+var audioNode = null;    // This example uses one audio node only.
+var content = null;      // The <div> displaying everything.
+var pitchShift = 0;      // The current pitch shift value.
+var currentPath = null;
+var pbPerc = null;
+
+function changePitchShift(e) {
+    // limiting the new pitch shift value
+    let value = parseInt(e.target.value);
+    pitchShift += value;
+    if (pitchShift < -12) pitchShift = -12; else if (pitchShift > 12) pitchShift = 12;
```

Comment:
Can we have more "Javascript" here? Such as:

`pitchShift = Math.min(12, Math.max(-12, pitchShift + parseInt(e.target.value)));`